### DISCUSSION TITLE
How to set the struct member color for C language

### DISCUSSION THREAD

**Discussion by cfcy** [NvChad/NvChad#2328] [Q&A] [answered]
Hi, I used the Solarized Dark corlor scheme. And I found the struct members and function paramters are all shown in red. Does anyone know how to change it? Thank you. I have overridden the Identifier and Statement group to white. But it did not take effect.

![1](https://github.com/NvChad/NvChad/assets/80516475/2e67409a-906a-4750-ae1b-c96bba6332ee)
![2](https://github.com/NvChad/NvChad/assets/80516475/b7a20a4d-01af-418b-b58b-1c83f966b7d1)

**Comment by siduck** [comment, marked as answer]
@cfcy check with Inspect command & InspectTree, if the hlgroup isnt defined by nvchad then add in hl_add option

**Reply by cfcy** [reply]
Hi, siduck, 
I am a newcomer to neovim. I am sure I did it write. Please point it out if I am wrong.
1. I navigated to a struct member and run command `:Inpsect`
![image](https://github.com/NvChad/NvChad/assets/80516475/2ff4e788-012d-4e00-9415-28225cbf6502)
2. Then run command: `InspectTree`
![image](https://github.com/NvChad/NvChad/assets/80516475/c4e737ff-15ba-40b8-90f6-e3a59ca12051)
3. I have changed the Identifier and field_identifier to white. 
![image](https://github.com/NvChad/NvChad/assets/80516475/e57ee1fd-e428-4a5e-82a4-39fdd9aae40c)

**Reply by siduck** [reply]
@cfcy use hl_overide then https://nvchad.com/docs/config/theming

**Reply by cfcy** [reply]
Hi, siduck

Finally I found the group `["@property"]` control the struct member color. Thank you for your prompt response.

**Comment by cfcy** [comment]
By adding ["@property"] to M.override={} in highlights.lua file solved this problem.
``` 
M.override = {
["@property"] = { fg = "white", },
}
```
